l10n_ar_afipws/views/res_partner.xml

```diff
@@ -8,7 +8,7 @@
             <group name="accounting_entries" position="after">
                 <group name="miPyme" string="Mi pyme">
                     <field name="mipyme_required"/>
-                    <field name="mipyme_from_amount" attrs="{'invisible':[('mipyme_required','=', False)]}"/>
+                    <field name="mipyme_from_amount" invisible="mipyme_required == False"/>
```

Comment:
Por convención se usa la negacion de la expresión  en ves del == False.  quedaría algo así

`<field name="mipyme_from_amount" invisible="not mipyme_required"/>`